mods/ts/rules/shared-support.yaml

```diff
@@ -9,6 +9,7 @@ NAPULS:
 		Queue: Defense
 		BuildPaletteOrder: 90
 		Prerequisites: radar
+		BuildLimit: 1
```

Comment:
https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs#L230

It chooses one of those support powers to get the order targeter. I'd have to search for all other emp cannons somewhere. Which I haven't done in this pr (yet). Which means that you can't fire at targets in range of any of your emp cannons, but of one of those.